R/comp_corr.R

```diff
@@ -1,11 +1,12 @@
 #' Computing corr
 #'
-#' This function computes the correlation upon picking the best lag and
-#' adjusting the data for lags
+#' This function computes the weighted correlation with penalty (no penalty if not lagged) after the lags
```

Comment:
This description sounds good.  I suggest phrasing it as `after the lags are applied` and `using the functions prep.data()`.

Reply:
Fixed.